knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml

```diff
@@ -794,7 +794,7 @@ data:
           "thresholds": "",
           "timeFrom": null,
           "timeShift": null,
-          "title": "Trigger (broker-filter): Success Rate  (2xx Event, fraction rate per minute)",
+          "title": "Trigger (broker-filter): Success Rate  (2xx Event, fraction rate, over 1m window)",
```

Comment:
@skonto Another instance of double spacing between "Success Rate" and "(2xx...".